Dockerfile

```diff
@@ -25,7 +25,7 @@ COPY . .
 ARG BUILD_ENV
 
 # Build client bundle and prepare for Server-Side Rendering
-RUN npm run build:ssr:${BUILD_ENV}
+RUN npm run build:${BUILD_ENV}
```

Comment:
My understanding from comparing new project boilerplate and searching the web is that there isn't a separate `ssr` build any more, so I tried to make our script names less confusing accordingly.